spec/view_cooks_spec.rb

```diff
@@ -0,0 +1,48 @@
+require "rails_helper"
+
+feature "view cooks" do
+  scenario "user can see homepage" do
+    visit root_path
+    expect(page).to have_content "Batch-it!"
+  end
+
+  scenario "user can visit new cook form from homepage" do
+    visit root_path
+    click_button "I am a cook"
+    visit new_post_path
+    expect(page).to have_content "New Cook"
+  end
+
+  scenario "user can create a new cook post" do
+    visit new_post_path
+
+    fill_in :cook, with: "Test cook"
+    fill_in "post_title", with: "Test cuisine"
+    fill_in "post_content", with: "Test details describing the delicious food I cook"
+    click_button "Create Post"
+
+    #How do I tell RSpec which page to navigate to, given the url changes depending on the post creating eg /posts/12?
+    visit posts_path
```

Comment:
You should already be on the correct page, you shouldn't have to `visit` here.

Reply:
Ok great, that's great to understand